kong/plugins/key-auth/migrations/postgres.lua

```diff
@@ -44,4 +44,13 @@ return {
     end,
     down = function(_, _, dao) end  -- not implemented
   },
+  {
+    name = "2018-04-26_key-auth_expires_in",
+    up = [[
+      ALTER TABLE keyauth_credentials ADD COLUMN expires_in int;
```

Comment:
Should the default be set in the schema? (`SET DEFAULT 0`)
Should negative values be accepted? (`CHECK (expires_in > 0)`)

Reply:
Setting a default value would cause a deviation in consistency between Postgres and Cassandra since Cassandra does not support default values on columns.